src/main/java/net/minecraftforge/client/ArmorOverlayHandler.java

```diff
@@ -0,0 +1,96 @@
+package net.minecraftforge.client;
+
+import net.minecraft.client.model.ModelBase;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.texture.TextureManager;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.ForgeModContainer;
+
+/**
+ * Offshore handler for the Armors' extra passes. Specifically for reducing patch size & maintaining readability.
+ * */
+public class ArmorOverlayHandler
+{
+
+    private static final java.util.Map<net.minecraftforge.fml.common.registry.RegistryDelegate<Item>, IArmorOverlay> armorOverlayMap = com.google.common.collect.Maps.newHashMap();
+
+    /**
+     * Number of passes to be run.
+     * */
+    private static final byte NUM_PASS = 2;
+
+    private ArmorOverlayHandler(){}
+
+    /**
+     * Handles the execution of overlays.
+     * */
+    public static boolean run(TextureManager textureManager, ItemStack stack, EntityLivingBase wearer, float partialTicks, EntityEquipmentSlot slot, ModelBase model, float swing, float swingScale, float headYaw, float headPitch, float scale)
```

Comment:
Definitely can put a better method name. I was being minimalistic since these were the methods used for the patches. But a handful of characters for clarity's sake is a fart in the wind really